src/components/Header.jsx

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from 'react'
+import Calender from './Calender'
+import axios from 'axios'
+
+const api = axios.create({
+  baseURL: `http://localhost:3001/`
+})
+
+
+export default class Header extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      value: '',
+      toDos: [],
+      date: '',
+    };
+  }
+
+
+  handleChange = (event) => {
+    this.setState({ value: event.target.value });
+  }
+  
+  handleSubmit = (event) => {
+    api.post('toDos', {toDoText: this.state.value}).then(resp => {
```

Comment:
do not use .then
follow try catch async await! 
![image](https://user-images.githubusercontent.com/22271897/121679349-f90b9f80-cad7-11eb-967a-da7d54f853b6.png)